class.jetpack.php

```diff
@@ -492,6 +492,11 @@ private function __construct() {
 			wp_schedule_event( time(), 'hourly', 'jetpack_clean_nonces' );
 		}
 
+		add_action( 'jetpack_clean_exports', array( 'Jetpack', 'clean_exports' ) );
+		if ( ! wp_next_scheduled( 'jetpack_clean_exports' ) ) {
+			wp_schedule_single_event( time() + 600, 'jetpack_clean_exports' );
```

Comment:
Rather than using `600` here and below, it's likely more readable to use `10 * MINUTE_IN_SECONDS` (Core [added these constants a couple years back](https://github.com/WordPress/WordPress/blob/493f76a3d2ef8c030ab5dcd4333f9a401208f534/wp-includes/default-constants.php#L107-L126), and they're much easier to parse when skimming code)